A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -1944,13 +1950,19 @@ class FMCMainDisplay extends BaseAirliners {
         return true;
     }
 
-    getTransitionAltitude() {
-        if (isFinite(this.transitionAltitude)) {
-            return this.transitionAltitude;
-        }
-
-        // TODO fetch this from the nav database once we have it in future
-        return 10000;
+    async getTransitionAltitude(departICAO) {
+        await NXApi.getAirport(departICAO)
```

Comment:
**question**: Why do you use `async` and `await` here? What is your intended effect?

Reply:
It only call when airport is change. So need to wait response

Reply:
Please read up on [how async/await works](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Asynchronous/Async_await) in JavaScript. It isn't necessary to use it here, or alternatively you can replace the promise code with it.